nntrainer/app_context.h

```diff
@@ -33,6 +33,12 @@
 
 #include <nntrainer_error.h>
 
+#ifdef _WIN32
+#define ML_API __declspec(dllexport)
```

Comment:
Can you refer to ```nntrainer/tensor/tensor.h``` instead of redefining it? I understand it needs to be "re"-defined for API, but, this is internal header.